GravatarApp/Share/ShareHeaderView.swift

```diff
@@ -0,0 +1,126 @@
+import Gravatar
+import SwiftUI
+
+struct ShareHeaderView: View {
+    @EnvironmentObject var userSession: UserSession
+
+    @Binding var forceRefresh: Bool
+
+    let qrGenerator: QRGenerator
+    @State var qrImage: UIImage?
+    @State var contentHeight: CGFloat = 0
+    @State var contentWidth: CGFloat = 0
+    @Binding var safeAreaInsets: EdgeInsets
+    let windowWidth: CGFloat
+    let maxHeight: CGFloat
+
+    private var imageURL: URL? {
+        AvatarURL(
+            with: .hashID(userSession.profile.hash),
+            options: .init(preferredSize: .pixels(Int(contentWidth / 2)))
+        )?.url
+    }
+
+    init(
+        forceRefresh: Binding<Bool>,
+        profile: Profile,
+        safeAreaInsets: Binding<EdgeInsets>,
+        width: CGFloat,
+        maxHeight: CGFloat
+    ) {
+        self._forceRefresh = forceRefresh
+        self.qrGenerator = QRGenerator(profile: profile)
+        self._safeAreaInsets = safeAreaInsets
+        self.windowWidth = width
+        self.maxHeight = maxHeight
+    }
+
+    var body: some View {
+        ZStack(alignment: .top) {
+            backgroundImageView(
+                imageURL: imageURL
+            )
+            .frame(height: contentHeight)
+            .clipped()
+            backgroundMaterial
+
+            VStack(spacing: 16) {
+                Text("Quickly share your public contact info by letting other scan this QR code.")
+                    .multilineTextAlignment(.center)
+                    .foregroundStyle(Color.white)
+                    .frame(width: windowWidth)
+                    .padding()
```

Comment:
This was supposed to put a padding around this text but I don't see it in the UI. Can you maybe poke a few places to see why this is happening? 🤔  

<img src="https://github.com/user-attachments/assets/2a2d4a2c-f573-4ab8-bb45-20f260de199e" width=300/>